web/src/routes/SurveyPages/SurveyListPage.tsx

```diff
@@ -1,5 +1,201 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+
+import { AxiosResponse } from 'axios';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import axios from '../../api/axios';
+import requests from '../../api/request';
+import Header from '../../components/Header';
+import { useTheme } from '../../hooks/useTheme';
+
+const Container = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background-color: ${(props) => props.theme.colors.container};
+`;
+
+const ListContainer = styled.div`
```

Comment:
So, looks like this is a table. Avoid using all tags with `<div>`s. Select proper tags for its component!

`ListContainer` should be more like `<table>`.

Reply:
@sanghyun1128  Please resolve this first